tests/integration/mixer/scenarios_test.go

```diff
@@ -120,46 +123,48 @@ func testMetric(t *testing.T, ctx framework.TestContext, label string, labelValu
 
 // Port of TestTcpMetric
 func TestTcpMetric(t *testing.T) {
-	t.Skip("https://github.com/istio/istio/issues/11430")
-	ctx := framework.NewContext(t)
-	defer ctx.Done(t)
-
-	ctx.RequireOrSkip(t, environment.Kube)
-
-	d := bookinfo.DeployOrFail(t, ctx, bookinfo.BookInfo)
-	_ = bookinfo.DeployOrFail(t, ctx, bookinfo.BookinfoRatingsv2)
-	_ = bookinfo.DeployOrFail(t, ctx, bookinfo.BookinfoDb)
+	framework.Run(t, func(ctx framework.TestContext) {
+		ctx.RequireOrSkip(t, environment.Kube)
 
-	g := galley.NewOrFail(t, ctx, galley.Config{})
-	_ = mixer.NewOrFail(t, ctx, mixer.Config{Galley: g})
+		_ = bookinfo.DeployOrFail(t, ctx, bookinfo.Config{Namespace: "istio-bookinfo", Cfg: bookinfo.BookInfo})
+		_ = bookinfo.DeployOrFail(t, ctx, bookinfo.Config{Namespace: "istio-bookinfo", Cfg: bookinfo.BookinfoRatingsv2})
+		_ = bookinfo.DeployOrFail(t, ctx, bookinfo.Config{Namespace: "istio-bookinfo", Cfg: bookinfo.BookinfoDb})
 
-	g.ApplyConfigOrFail(
-		t,
-		d.Namespace(),
-		bookinfo.NetworkingBookinfoGateway.LoadOrFail(t),
-		bookinfo.GetDestinationRuleConfigFile(t, ctx).LoadOrFail(t),
-		bookinfo.NetworkingVirtualServiceAllV1.LoadOrFail(t),
-		bookinfo.NetworkingTCPDbRule.LoadOrFail(t))
+		g := galley.NewOrFail(t, ctx, galley.Config{})
+		_ = mixer.NewOrFail(t, ctx, mixer.Config{Galley: g})
 
-	prom := prometheus.NewOrFail(t, ctx)
-	ing := ingress.NewOrFail(t, ctx, ingress.Config{Istio: ist})
-
-	err := visitProductPage(ing, 30*time.Second, 200, t)
-	if err != nil {
-		t.Fatalf("unable to retrieve 200 from product page: %v", err)
-	}
+		ns, err := namespace.Claim(ctx, "default")
+		if err != nil {
+			t.Fatalf("Could not get default Namespace:%s; err:%v", "default", err)
+		}
+		g.ApplyConfigOrFail(
+			t,
+			ns,
+			strings.Replace(bookinfo.NetworkingBookinfoGateway.LoadOrFail(t), "default", "istio-bookinfo", -1),
```

Comment:
This might accidentally replace word default that is not a namespace.

Reply:
Yes, it's an assumption.. the only other way I think is to create a copy of these files for testing..
that might be cleaner and not affect public docs then?

Reply:
The only stable that works from our experience was not to have any mention of the namespace in the config. Then the tooling can put the config in the right namespace. I don't know if that's something feasible with the current APIs but it looks like that before we did not have namespace and now we do.

Reply:
yeah that's what am seeing now...

@andraxylia : can you please help comment on this?

Also, with change in these files.. I will have to make changes to other test files too.. some other tests are failing now..will do it once we are fine with the approach in this PR..

Reply:
Created a copy of the files for test...